### ISSUE TITLE
Exemplo

### ISSUE THREAD

**Issue by roneydourados** [academiadocodigo/horse-cachecontrol#2]
Poderia acrescentar na documentação um exemplo de como usar por gentileza?

**Comment by viniciussanchez** [comment]
Não tenho certeza, mas acho que o @bittencourtthulio ou o pessoal da Academia do código fizeram vídeo sobre isso.

**Comment by viniciussanchez** [comment]
https://academiadocodigo.com.br/passaporte-delphi/boas-praticas-para-servidores-restful-com-horse/

Na verdade ele fez esse treinamento... mas não cheguei a ver o funcionamento disso...

**Comment by roneydourados** [comment]
Olá Vinicius muito obrigado pelo feedback, então infelizmente para isso tem que comprar o treinamento né ? Ajudaria demais se colocassem no readme ali como utilizar a lib. Grato!

**Comment by viniciussanchez** [comment]
Sim... infelizmente eu também não sei como funciona, mas solicitei pra eles algo... vamos ver

**Comment by roneydourados** [comment]
Maravilha Vinicius, vamos aguardar. Obrigado!

**Comment by viniciussanchez** [comment]
@roneydourados teve algum retorno? Talvez o @alepmedeiros possa ajudar em algo...

**Comment by roneydourados** [comment]
Opa, @viniciussanchez então não cheguei a entrar em contato com eles, na ultima mensagem você mencionou que solicitou para pessoal lá. Meu caso mudei de estratégia, infelizmente tive que abandonar o delphi, fiz o api em NodeJS.

**Comment by viniciussanchez** [comment]
Entendi... uma pena...
